### PR TITLE
Added the deprecated lowercase v8 API to codecov ignore list

### PR DESCRIPTION
**Pull request by Dillon-Brown**
## Description

The old v8 API is deprecated and we don't run any of the tests for it in Travis anymore. 

## How To Test This

1. See Travis.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [X] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

File: .codecov.yml [hunk 1]
```diff
@@ -59,3 +59,4 @@ ignore:
   - "install/demoData.en_us.php"
   - "include/tcpdf/.*"
   - "modules/AOR_Charts/lib/.*"
+  - "lib/API/.*"
```
Comment (connorshea) [inline .codecov.yml]: Should maybe add a comment explaining why this is ignored?

**Comment by Mac-Rae** [comment]
Assessed :+1:

**Comment by pgorod** [comment]
I don't want to sound picky, and it's just a comment, but it's not intuitive for everybody that `v8` is not the same thing as `V8`. So people are going to read that and think that their shiny new API was deprecated...

So maybe something like 

> # The old lowercase v8 API is deprecated/unsupported and the CI doesn't run the tests for it. The new API uses an upper-case V.

**Comment by Dillon-Brown** [comment]
@pgorod Agreed.

**Pull request merged**